evaporate.js

```diff
@@ -14,25 +14,27 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
 /***************************************************************************************************
 *                                                                                                  *
-*  version 0.0.2                                                                                  *
-*                                                                                                  *
-*  TODO:                                                                                           *
-*       calculate MD5s and send with PUTs                                                          *
-*       post eTags to application server to allow resumability after client-side crash/restart      *
-*                                                                                                  *
+*  version 1.0.0                                                                                   *
 *                                                                                                  *
 ***************************************************************************************************/
 
 (function() {
-
-  var Evaporate = function(config){
-
-     this.supported = !((typeof(File)=='undefined') ||
-        (typeof(Blob)=='undefined') ||
-        !(!!Blob.prototype.webkitSlice || !!Blob.prototype.mozSlice || Blob.prototype.slice) ||
+   var FAR_FUTURE = new Date('2060-10-22');
+
+  var Evaporate = function (config) {
+
+     this.supported = !(
+        typeof File === 'undefined' ||
+        typeof Blob === 'undefined' ||
+        typeof (
+           Blob.prototype.webkitSlice ||
+           Blob.prototype.mozSlice ||
+           Blob.prototype.slice ||
+           (config.computeContentMd5 && Blob.prototype.readAsBinaryString) === 'undefined') ||
```

Comment:
Regarding `readAsBinaryString`. I did not encounter that warning and was following what has worked and is in the code. The latest 'supported' check corrects the reference from Blob to FileReader.

Let me replace this with readAsArrayBuffer and see where it goes.